frontend/src/components/eMIB/Email.jsx

```diff
@@ -126,6 +132,26 @@ class Email extends Component {
         </div>
         <hr style={styles.dataBodyDivider} />
         <div>{email.body}</div>
+        <div>
+          {emailActions.map((email, id) => {
+            return (
+              <CollapsingItemContainer
+                key={id}
+                icon={ICON_TYPE.email}
+                title={"Email response"}
```

Comment:
You should maybe add a **TODO** here, saying that we need to put a dynamic title generator or something like that, instead of hard coding a title.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/181/commits/6c84cd2ee4658565145d4fc5bdc2d47a23477060